src/components/calculator/Calculator.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './calculator.scss';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends Component {
```

Comment:
- [ ] [OPTIONAL] rather than disabling the linter you can use React.PureComponent, and it should resolve the issue. :+1: